include/IVideoDriver.h

```diff
@@ -227,12 +227,6 @@ namespace video
 		Usually, there is no need to call this method. */
 		virtual void OnResize(const core::dimension2d<uint32_t>& size) =0;
```

Comment:
should think about how to handle window resizes (callback, virtual method, event?)

then nuke this entire file